android-plugin/RULES.md

```diff
@@ -0,0 +1,38 @@
+# Android-specific Rules
+
+Android-specific rules rely on a multi-scope scanning, including Java source files, Xml files, Gradle files, and more generally the project structure.
+
+### 🍃 Environment
+
+The set of rules comes from the detailed [Energy Smells catalog](https://olegoaer.perso.univ-pau.fr/android-energy-smells/). 3/4 of them have been already implemented in the plugin. Table of unimplemented rules below:
+
+| # | **Rule Name**      |     **Scanner**     |      **Observation**     |
+|---|:----------------|:-------------:|:-------------:|
+| ELEA001 | Everlasting Service        | Java | Requires `PostProjectAnalysisTask()` callback |
+| EBOT004 | Uncached Data Reception       | Java | Requires `PostProjectAnalysisTask()` callback |
+| ESOB003/ESOB004 | Dark UI      | Java, Xml | Partially implemented (e.g. [1](./src/main/java/io/ecocode/xml/checks/sobriety/DarkUIBrightColorsXmlRule.java), [2](./src/main/java/io/ecocode/xml/checks/sobriety/DarkUIThemeXmlRule.java)) |
```

Comment:
For this kind of partialy implemented rule, we should give more precision (what is done and what we expect)

Reply:
I don't remember why this rule wasn't completed. Any idea @Jberque?